extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -27,6 +28,10 @@ amp-sticky-ad {
   background-color: rgba(255,255,255,0.5);
 }
 
+amp-sticky-ad.visible {
```

Comment:
I use `setStyles` again and delete this css class.
Since the layout type is `nodisplay`, I don't worry about user setting `amp-sticky-ad`  `visibility:visible` now. Thus I didn't put `!important` on `visibility: hidden` and was able to use `setStyles` function.